cogs/bot.py

```diff
@@ -452,16 +452,9 @@ def cog_unload(self):
             self.post_dbl.cancel()
             self.remind_votes.cancel()
 
-    @commands.command()
-    @commands.has_permissions(manage_messages=True)
-    async def cleanup(self, ctx, search=100):
-        """Cleans up the bot's messages from the channel."""
-
-        def check(m):
-            return m.author == ctx.me or m.content.startswith(ctx.prefix)
-
+    async def run_purge(self, ctx, limit, check):
```

Comment:
I don't really see a difference between the two implementations though... It looks like they are running the same code, just with two functions instead of one. Correct me if I am wrong.